components/search_results/search_results.jsx

```diff
@@ -197,23 +196,32 @@ class SearchResults extends React.Component {
             );
         } else if (this.props.isFlaggedPosts && noResults) {
             ctls = (
-                <div className='sidebar--right__subheader a11y__section'>
-                    <FlagPostSearchHint
-                        dataRetentionEnableMessageDeletion={this.props.dataRetentionEnableMessageDeletion}
-                        dataRetentionMessageRetentionDays={this.props.dataRetentionMessageRetentionDays}
+                <div
+                    className={classNames(['sidebar--right__subheader a11y__section',
+                        {'sidebar-expanded': this.props.isSideBarExpanded && noResults,
+                        }])}
+                >
+                    <NoResultsIndicator
+                        variant={NoResultsVariant.FlaggedPosts}
+                        subtitleValues={{icon: <FlagIcon className='icon  no-results__icon'/>,
+                        }}
                     />
                 </div>
             );
         } else if (this.props.isPinnedPosts && noResults) {
             ctls = (
-                <div className='sidebar--right__subheader a11y__section'>
-                    <PinPostSearchHint
-                        dataRetentionEnableMessageDeletion={this.props.dataRetentionEnableMessageDeletion}
-                        dataRetentionMessageRetentionDays={this.props.dataRetentionMessageRetentionDays}
+                <div
+                    className={classNames(['sidebar--right__subheader a11y__section',
+                        {'sidebar-expanded': this.props.isSideBarExpanded && noResults,
+                        }])}
+                >
+                    <NoResultsIndicator
+                        variant={NoResultsVariant.PinnedPosts}
+                        subtitleValues={{text: <strong>{'Pin to Channel'}</strong>}}
```

Comment:
is this supposed to be internationalized?

Reply:
I am not sure for now.